image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
+Allows for mapping random access to the references in the image-layout, and needing only to read from the `oci-layout` once to build this map of references.
+
+**index** *array of objects*, OPTIONAL
+
+  Provides a single access to top-level information about the layout.
+  Each object has the following properties:
+
+  - **ref** *string*, REQUIRED
+  
+    The name of the ref (correspond JSON descriptor found in `./refs/`)
+
+  - **annotations** *string-string map*, OPTIONAL
+
+    This field is under the same constaints as [manifest `annotations`](manifest.md#image-manifest-property-descriptions)
```

Comment:
oh, it's just here, there and everywhere. The further dilution of topics never seems to have the intention to arrive at a conclusion, only to further add items to come to agreement on.